website/documentation/getting-started/shoots/_index.md

```diff
@@ -0,0 +1,124 @@
+
+In case you have specific requirements for the cluster internal DNS, Gardener offers a plugin mechanism for custom core DNS rules or optimization with node-local DNS. For more information, see [Custom DNS Configuration](https://github.com/gardener/gardener/blob/master/docs/usage/custom-dns-config.md) and [NodeLocalDNS Configuration](https://github.com/gardener/gardener/blob/master/docs/usage/node-local-dns.md).
+
+Another category of configuration options is dedicated to the nodes and the infrastructure they are running within. Every provider has their own perks and some of them are exposed. Check the detailed documentation of the relevant infrastructure provider extension.
+
+You can fine-tune the cluster-autoscaler or help the kubelet to cope better with your workload.
+
+## Worker Pools
+
+There are a couple of ways to configure a worker pool. One of them is to set everything in the Gardener dashboard. However, only a subset of options is presented there.
+
+![](./images/worker-pools-1.png)
+
+A slightly more complex way is to set the configuration through the yaml file itself.
+
+![](./images/worker-pools-2.png)
+
+You can configure much more, for example, the health timeout after which a machine is getting replaced. For more options, see the [Worker](https://github.com/gardener/gardener/blob/master/docs/api-reference/core.md#worker) API reference.
+
+## How to Change Things
+
+Since a shoot is just another Kubernetes resource, changes can be applied via kubectl. For convenience, the basic settings are configurable via the dashboard's UI. It also has a "yaml" tab where you can alter all of the shoot's specification. Once applied, the cluster will reconcile eventually and your changes become active (or cause an error).
+
+![](./images/change-things.png)
+
+## Immutability in a Shoot
+
+However, not everything can be changed after the creation of the shoot. For example, it is not possible to move a shoot to a different infrastructure account. This is mainly rooted in the fact that discs and network resources are bound to your account. 
+
+Once created, most of the network aspects of a cluster become immutable. On an infrastructure level the VPC cannot be changed and on a cluster level things like the pod / service cidr ranges, together with the nodeCIDRmask, are set for the lifetime of the cluster.
+
+![](./images/immutablity.png)
+
+Some other things can be changed, but not reverted. While it is possible to add more zones to a cluster on an infrastructure level (assuming that an appropriate CIDR range is available), removing zones is not supported. Similarly, upgrading Kubernetes versions is comparable to a one-way ticket. As of now, Kubernetes does not support downgrading. Lastly, the HA setting of the control plane is immutable once specified.
```

Comment:
```suggestion
Some other things can be changed, but not reverted. While it is possible to add more zones to a cluster on an infrastructure level (assuming that an appropriate CIDR range is available), removing zones is not supported. Similarly, upgrading Kubernetes versions is comparable to a one-way ticket. As of now, Kubernetes does not support downgrading. Lastly, the HA setting of the control plane is immutable once specified.

## Crazy Botany
Since remembering all these options can be quite challenging, here is very helpful resource - an [example shoot](https://github.com/gardener/gardener/blob/master/example/90-shoot.yaml) with all the latest options 🎉 
```